src/tests/ftest/util/mpio_test_base.py

```diff
@@ -45,21 +48,12 @@ def setUp(self):
         super(LlnlMpi4pyHdf5, self).setUp()
 
         try:
-            # parameters used in pool create
-            createmode = self.params.get("mode", '/run/pool/createmode/*/')
-            createuid = os.geteuid()
-            creategid = os.getegid()
-            createsetid = self.params.get("setname", '/run/pool/createset/')
-            createsize = self.params.get("size", '/run/pool/createsize/')
-            self.createsvc = self.params.get("svcn", '/run/pool/createsvc/')
-
             # initialize a python pool object then create the underlying
-            # daos storage
-            self.pool = DaosPool(self.context)
-            self.pool.create(createmode, createuid, creategid,
-                             createsize, createsetid, None, None,
-                             self.createsvc)
-        except (DaosApiError) as excep:
+            self.pool = TestPool(
+                self.context, dmg_command=self.get_dmg_command())
+            self.pool.get_params(self)
+            self.pool.create()
+        except (CommandFailure) as excep:
```

Comment:
The TestPool.create() has a decorator to catch the `CommandFailure` and it will instead raise a `TestFail` exception.  So lines 50 & 56-57 can simply be removed.